src/software/jetson_nano/primitive_executor.cpp

```diff
@@ -34,6 +34,11 @@ void PrimitiveExecutor::clearCurrentPrimitive()
     current_primitive_.Clear();
 }
 
+void PrimitiveExecutor::setStopPrimitive()
+{
+    current_primitive_ = *createStopPrimitive(true);
```

Comment:
@LiCody @itsarune @AdenC123 This seems to be unused right now, but do you know what the intended difference between `TbotsProto::StopPrimitive::BRAKE` and `TbotsProto::StopPrimitive::COAST` is? Does `COAST` keep the velocity, or does it slowly decelerate to a stop rather than breaking hard. Trying to see whether we should be passing `true` or `false` here. If there is no plan on using `TbotsProto::StopPrimitive`, I think we should probably remove it.

Reply:
I think the intention was that brake would immediately stop motors, while coast would gradually let friction stop the wheels

Reply:
I think we can get rid of it, and probably merge it with EstopPrimitive and StopPrimitive

Reply:
Removed BRAKE, COAST and EstopPrimitive, they should all have the same behaviour now.